cmd/minishift/cmd/start.go

```diff
@@ -419,6 +425,11 @@ func initClusterUpFlags() *flag.FlagSet {
 	clusterUpFlagSet.AddFlag(cmdutil.HttpProxyFlag)
 	clusterUpFlagSet.AddFlag(cmdutil.HttpsProxyFlag)
 
+	if future {
```

Comment:
why not used 'FlagSet.MarkHidden(name string)' to just hide the flag instead of not adding it at all?